histomicstk/segmentation/label/trace_boundary.py

```diff
@@ -1,6 +1,6 @@
 import numpy as np
 import ctypes
-from .isbf import isbf
+from .trace_object_boundary import trace_object_boundary
 
 
 def trace_boundary(Mask, Connectivity=4, XStart=None, YStart=None,
```

Comment:
Also, i tried the above function `get_label_contours` on a label mask with  over 1000 objects that @dgutman sent me with TraceBounds and trace_boundary.

When i used TraceBounds it took 160 seconds. When i used trace_boundary instead it tool 2400 seconds.

I think the main reason for this is the [rot90](https://github.com/DigitalSlideArchive/HistomicsTK/blob/0a7156c81f28f872916e7a2b29cb894897eabada/histomicstk/segmentation/label/trace_boundary_cpp.cpp#L15) which computes rotated versions of the input image is being called for each and every object. We only need to call this once.

Also, another reason is that the loop over objects is in python.